utils/queryApiGateway.ts

```diff
@@ -112,6 +113,51 @@ export function getUniqueNumber(req: IncomingMessage): string {
   return req.headers[idHeaderName.toLowerCase()] as string
 }
 
+/**
+ * Check if the API responded with a null response body
+ *
+ * We do not expect any of these in production, in theory.
+ */
+export function reponseIsNull(apiBody: Claim): boolean {
+  const response: Claim = JSON.parse(JSON.stringify(apiBody)) as Claim
```

Comment:
I think you should do this instead:
```suggestion
  const response = extractJSON(apiBody)
```
But you should do it inside a `try catch` block (either in this function or the function calling this function) because JSON parsing can throw errors

Reply:
(I will add a comment explaining this syntax)

Reply:
```suggestion
  // This is a trick to make a deep-copy of a JSON object
  const response: Claim = JSON.parse(JSON.stringify(apiBody)) as Claim
```